woocommerce/woocommerce
# Add PR numbers to changelog entries on code freeze process

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/33522

This PR adds the merged PR number to the end of the changelog entries generated during the code freeze automation process.

### How to test the changes in this Pull Request:

1) Fork this repository.
2) Enable issues in your fork from `https://github.com/YOURORG/woocommerce/settings`
3) Enable actions in your fork from `https://github.com/YOURORG/woocommerce/settings/actions`. Also, make sure workflows have read and write permissions and the ability to create and approve PRs.
4) Enable the scheduled workflow from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml`.
6) Merge this PR into `trunk` of your new fork.
7) In your fork, create a release `6.7.0`.
8) If there aren't already changes in `plugins/woocommerce/changelog`, in trunk, create several changes in `trunk` that add change files to the changelog dir (whether via PRs or just manually adding change files).
9) Run the action manually from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml` with "2022-08-02" as the value for time override. Disable the slack ping for the manual trigger.
10) The script should run, create `release/6.8` from trunk, create a `6.9.0` milestone, generate the changelog, create a PR against `trunk` that removes the change files, and a PR against `release/6.8` that updates the changelog and removes the change files.
11) Check the created PR against `release/6.8` and ensure you see a changelog.txt with PR entries and the PR numbers at the end.

### Other information:




### FOR PR REVIEWER ONLY:
